Sources_v3/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -364,6 +364,47 @@ public extension _CurrentChatUserController {
         }
     }
     
+    /// Updates the current user.
+    ///
+    /// By default all data is `nil`, and it won't be updated unless a value is provided.
+    ///
+    /// - Parameters:
+    ///   - name: Optionally provide a new name to be updated.
+    ///   - imageURL: Optionally provide a new image to be updated.
+    ///   - userExtraData: Optionally provide new user extra data to be updated.
+    ///   - completion: Called when user is successfuly updated, or with error.
+    func updateUser(
```

Comment:
`updateUser` -> `updateUserData` ? Update users can be understood like "changing" the actual logged-in user.